tasks/main.yml

```diff
@@ -1,202 +1,31 @@
 ---
-- name: Forced facts gathering
-  setup:
-  when: ansible_os_family is not defined
-  tags:
-    - cartridge-instances
-    - cartridge-replicasets
-    - cartridge-config
 
-- name: Check distribution
-  fail:
-    msg: 'Deploy to {{ ansible_os_family }} distributions is not supported yet'
-  when: ansible_os_family not in ["RedHat", "Debian"]
+- name: Forced Gathering Facts
```

Comment:
Why not in `tasks/facts`?

Reply:
Maybe it's better to move to `facts.yml`, yes

Reply:
It should be before config validation, but cartridge facts gathering should be after config validation